core/src/main/java/org/elasticsearch/common/network/NetworkUtils.java

```diff
@@ -227,7 +227,14 @@ public static boolean defaultReuseAddress() {
     
     /** Returns addresses for the given interface (it must be marked up) */
     static InetAddress[] getAddressesForInterface(String name) throws SocketException {
-        NetworkInterface intf = NetworkInterface.getByName(name);
+        NetworkInterface intf = null;
```

Comment:
wouldnt a stream using `getInterfaces().stream().filter().findFirst()` be nicer here?